packages/create-svelte/templates/skeleton/.meta.json

```diff
@@ -0,0 +1,3 @@
+{
+	"description": "Demo project"
```

Comment:
I can't find anywhere online that talks about `.meta.json`. Is this a VS Code thing?

Reply:
it's so that we can add a description of the templates when prompting

Reply:
ok, I missed that. it'd probably be nice if this wasn't added to the user's project. I didn't look at the code, so perhaps that's already happening

Reply:
it is not added to the user's project